plugins/BEdita/Core/config/Migrations/20221201092959_AddChildrenOrder.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * BEdita, API-first content management framework
+ * Copyright 2022 Atlas Srl, Chialab Srl
+ *
+ * This file is part of BEdita: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * See LICENSE.LGPL or <http://gnu.org/licenses/lgpl-3.0.html> for more details.
+ */
+
+use BEdita\Core\Utility\Resources;
+use Migrations\AbstractMigration;
+
+class AddChildrenOrder extends AbstractMigration
+{
+    protected $create = [
+        'property_types' => [
+            [
+                'name' => 'objects_order',
```

Comment:
I'd use `children_order` as property type name - this way we could handle the property automagically in the Manager app